# goal definition
bool start
---
# result definition
bool success

---
# feedback
mdr_perception_msgs/FaceBoundingBox[] bounding_boxes
int64 number_of_faces
float64[] center_of_face
string current_state
string text
